dotCMS/src/main/java/com/dotcms/rest/api/v1/workflow/WorkflowResource.java

```diff
@@ -75,7 +195,7 @@ protected WorkflowResource(final WorkflowHelper workflowHelper,
     @NoCache
     @Produces({MediaType.APPLICATION_JSON, "application/javascript"})
     public final Response findStepsByScheme(@Context final HttpServletRequest request,
-                                     @PathParam("schemeId") final String schemeId) {
+                                            @PathParam("schemeId") final String schemeId) {
 
         final InitDataObject initDataObject = this.webResource.init
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "initDataObject". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "initDataObject" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1481)

Reply:
Done